app/models/event/camp.rb

```diff
@@ -262,4 +266,15 @@ def may_become_sub_camp
     end
   end
 
+  def assert_contact_attrs_passed_on_to_supercamp_valid
+    contact_attrs_passed_on_to_supercamp.map(&:to_s).each do |a|
+      unless valid_contact_attr?(a)
+        errors.add(:base, :contact_attr_invalid, attribute: a)
```

Comment:
Braucht es hier noch eine Übersetzung? Für `contact_attr_passed_on_to_supercamp_hidden` existiert eine. Ich bin gerade unsicher, ob hier "Rails-Magie" greift oder nicht.

Reply:
Habs gerade getestet, scheint zu funktionieren:
> 'nickname2' ist kein gültiges Personen-Attribut

Dieselbe Message wird auch schon im core verwendet in `Event#assert_required_contact_attrs_valid`, und hat den key `activerecord.errors.models.event.attributes.base.contact_attr_invalid`. Rails-Magie in `errors.add` erkennt hier wohl, dass ein `Camp` auch ein `Event` ist und kann das somit auflösen.